docusign/docusign.lua

```diff
@@ -0,0 +1,79 @@
+--[[
+
+# [DocuSign API DOCUMENTATION](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](https://admindemo.docusign.com).
+
+## Input parameters
+
+* `username` - application user email used to authorize the call.
+* `password` - user password used to authorize the call
+* `integratorKey` - integration and authenticate with the DocuSign platform
+
+* api_endpoint - API endpoint URI, e.g. `user`.
```

Comment:
remove `user` example, there is no such endpoint, or refer to `login_information` as the real one.